README.md

```diff
@@ -58,97 +61,84 @@ There are 1 API for prediction class
-```
-
-#### `find_by`
-```ruby
-Datawow.[class].find_by({id: "_image_id", token: '_token'})
-```
 
-#### `all`
+Those method, it's a shortcut for calling of these class
 ```ruby
-Datawow.[class].all({token: '_token'})
+Datawow::ImageClosedQuestion
+Datawow::PhotoTag
+Datawow::ImageChoice
+Datawow::ImageMessage
+Datawow::NanameueHuman
+Datawow::VideoClassification
+Datawow::TextClosedQuestion
+Datawow::TextCategory
+Datawow::TextConversation
+Datawow::TextJa
+Datawow::Prediction
 ```
 ---
-## Nanameue with Consensus
-##### There are 2 ways to use the library
-
-Firstly, __dynamic token__. if you have multiple project to work with, You could call library by using module name
+## Avalabile method for APIs
+Every classes there are 3 functions that is `create`, `find_by` and `all`
 #### `create`
 ```ruby
-Datawow::NanameueHuman.new('_token').create({data: "image URL"})
+Datawow.[class].create(data: {})
 ```
 
 #### `find_by`
 ```ruby
-Datawow::NanameueHuman.new('_token').find_by({id: "_image_id"})
+Datawow.[class].find_by({id: "_image_id"})
 ```
 
 #### `all`
 ```ruby
-Datawow::NanameueHuman.new('_token').all({page: '_page', per_page: '_per_page'})
+Datawow.[class].all()
 ```
 
-or another way
 
-```ruby
-model = Datawow::NanameueHuman.new
-model.project_key = '_token'
-model.create()
-```
+---
+## Example
 
-Secondly, we have initiated each modules since the library being called, so you could call from package name
+Since the library being called. we have initiated each modules, so you could call from package that we provided [here](#class_explanation)
 ##### Setting project key
 ```ruby
 Datawow.project_key = '_token'
 ```
 ##### Start working with provided methods
 #### `create`
 ```ruby
-Datawow.nanameue_human.create({data: "image URL"})
+Datawow.[class].create({data: "image URL"})
 ```
 
 #### `find_by`
 ```ruby
-Datawow.nanameue_human.find_by({id: "_image_id"})
+Datawow.[class].find_by({id: "_image_id"})
 ```
 
 #### `all`
 ```ruby
-Datawow.nanameue_human.all({page: '_page', per_page: '_per_page'})
+Datawow.[class].all({page: '_page', per_page: '_per_page'})
 ```
 ---
 
-## Text AI Japanese
-##### Set your project key
-```ruby
-Datawow.project_key = '_token'
-```
 
-#### `create`
-```ruby
-Datawow.text_ja.create({ data: 'フーバーバズ', custom_id: 'custom_id', postback_method: 'GET', postback_url: 'https://datawow.io' })
-```
+## Dynamically token setting
+
+If you have many projects to work with. It's not proper to use above example. If you would like to create an object and change a token dynamically, We perfer to use these example
 
-#### `find_by`
 ```ruby
-Datawow.text_ja.find_by({ id: '_text_id' })
+Datawow::[class].new('_token').create({data: "image URL"})
 ```
-
-#### `all`
+or
 ```ruby
-Datawow.text_ja.all({page: '_page', per_page: '_per_page'})
+model = Datawow::[class].new
+model.project_key = '_token'
+model.create()
 ```
 
-# Setting default token
+For available method find in [calling APIs](#avalabile_method_for_apis) section
```

Comment:
For the available methods, see [calling APIs](#avalabile_method_for_apis) section